CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
   * after a scaling up, the operator triggers an auto-rebalancing for moving some of the existing partitions to the newly added brokers.
   * before scaling down, and if the brokers to remove are hosting partitions, the operator triggers an auto-rebalancing to these partitions off the brokers to make them free to be removed.
 * Strimzi Access Operator 0.1.0 added to the installation files and examples
+* Patch the StrimziPodSets before rolling the pods during CA certificate renewal.
```

Comment:
I am not sure this fix really deserves a line in the CHANGELOG. I mean, not sure what the users can do or learn from it.

Reply:
I added it because the users could run into the issue where a rolling update from a cert change is interrupted and then have the rolling update start again from the beginning. So I figured it's useful for us to be able to refer back to which version the fix went into incase people hit that

Reply:
> So I figured it's useful for us to be able to refer back to which version the fix went into incase people hit that

Well I guess we can find out going through the GitHub repo and fixed issues across a milestone but from my pov, the CHANGELOG is something user facing so what we put there should make any sense to users, not being helpful for us (if it can do both ... better!). @scholzj wdyt about this?

Reply:
I think having a changelog record is useful ... but we can probably make it better an more understandable to the users?

What about something like:
```
* Allow the rolling update to trust a new cluster CA (during Cluster CA key replacement) to continue where it left off before interuption instead of starting always from the beginning
```